charts/rstudio-library/templates/_rbac.tpl

```diff
@@ -12,7 +12,7 @@
 */ -}}
 {{- define "rstudio-library.rbac" -}}
 {{- $serviceAccountAnnotations := default (dict) .serviceAccountAnnotations }}
-{{- $serviceAccountCreate := default true .serviceAccountCreate }}
+{{- $serviceAccountCreate := eq .serviceAccountCreate false | ternary false true }}
```

Comment:
Ah interesting- I must admit that I'm surprised we are not making the type check here. I guess it allows us to have a cleaner error message if the downstream charts do the error check though.

Would you mind making a note in the comment above that this is required to be a boolean and will fail otherwise?

Reply:
Ah. I misunderstood how this was being done in the library.
It's somewhat of a dependency graph issue because the `serviceAccountCreate` value is actually set by the various charts that consume the library and passed to the named template.

Each of the charts consuming the library set this with a different values file entry: https://github.com/rstudio/helm/search?q=serviceAccountCreate&type=code

It might be better to move this validation into the library chart, and handle this better in #35 and #17

Reply:
This has been moved into the library chart.